data/language/dutch.txt

```diff
@@ -2341,13 +2341,13 @@ STR_2339    :Peseta (Pts)
 STR_2340    :Lire (L)
 STR_2341    :Guldens (fl.)
 STR_2342    :Kronen (kr)
-STR_2343    :Euro's ({EURO})
+STR_2343    :Euro's ({EURO}) €?
```

Comment:
Dat is mooi. Heb het puntje achter fl wel weggehaald trouwens, net zoals bij de andere munten is dit incorrect.